handlers/deploy.go

```diff
@@ -272,7 +199,7 @@ func makeDeploymentSpec(request requests.CreateFunctionRequest, existingSecrets
 							Name:  request.Service,
 							Image: request.Image,
 							Ports: []apiv1.ContainerPort{
-								{ContainerPort: int32(watchdogPort), Protocol: corev1.ProtocolTCP},
+								{ContainerPort: factory.Config.WatchdogPort, Protocol: corev1.ProtocolTCP},
```

Comment:
I guess `WatchdogPort` is the port of any server in our container.

What about `AltHTTPPort` or `HTTPPort` or `RuntimeHTTPPort`?